### PR TITLE
Homework 1.2

### PR DESCRIPTION
**Pull request by PimaUA**
Good day,

Please find below:
1.branch "homework 1.2" for pull request;
2.link to new added project to GitHub
https://github.com/PimaUA/Homework_1.1.git.

Best regards

### CODE REVIEW

File: src/com/hillel/Main.java [hunk 1]
```diff
@@ -1,8 +1,7 @@
 package com.hillel;
 
 public class Main {
-
     public static void main(String[] args) {
-        System.out.println("Hello Java");
+        System.out.println("Hello Java Pro!!!");
```
Comment (EugeniyaDzhu) [inline src/com/hillel/Main.java]: Please, make your changes in your separate package (name it with your name)

Comment (PimaUA) [reply src/com/hillel/Main.java]: The separate package was created.

File: src/com/hillel/Main.java [hunk 2]
```diff
@@ -1,8 +0,0 @@
-package com.hillel;
```
Comment (EugeniyaDzhu) [inline src/com/hillel/Main.java:1]: Why did you delete this class? That's why you have merge conflict. Restore this class from master (on your branch check master and Merge into Current)

File: src/DZ1Natom654/DZ1Natom654.java [hunk 3]
```diff
@@ -1,9 +0,0 @@
-package DZ1Natom654;
```
Comment (EugeniyaDzhu) [inline src/DZ1Natom654/DZ1Natom654.java]: And now you deleted all the rest files? :)
